src/gui/widgets/options_button.hpp

```diff
@@ -87,7 +87,19 @@ class options_button : public styled_widget
 		return values_.size();
 	}
 
-private:
+    /**
+     * Get the current state of the menu options.
+     *
+     * @returns        A mask specifying which options are selected
+     */
+	boost::dynamic_bitset<> get_toggle_states() const;
```

Comment:
This seems like something that belongs in multimenu_button, not in options_button.

Reply:
options (or really anything that inherits from it that isn't disabled) can have checkboxes (though in a later comment I'll advocate removing it anyway).

Reply:
Sure, options can have checkboxes, but the toggle_states system doesn't really make sense for a menu that has a few checkboxes mixed in with actions. It only really makes sense when _every_ option has a checkbox, I think. So that's why I think it belongs in multimenu_button.

Maybe there should be a function in options_button to check whether a given item is checked, however.